docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -670,49 +670,58 @@ paddlepaddle>=2.1
 
 | Operator Name in PaddlePaddle| Limitations|
 | :----------| :----------|
-| adpative_pool2d | The `NHWC` data_layout is not supported. |
 | arg_max | The `int32` output data_type is not supported. |
 | assign |  |
 | assign_value |  |
 | batch_norm |  |
+| bicubic_interp_v2 |  |
```

Comment:
I think we can just keep bicubic_interp and ignore the version info, as Paddle's users will only know them from api, instead of serialized protos.